ui/components/app/modals/import-token-exit-modal/index.scss

```diff
@@ -0,0 +1,4 @@
+
+.top-class {
+  top: 32vh;
```

Comment:
Nit: Where does this value come from?

Reply:
Nit: Where does this value come from?  Also, the `top-class` className is too generic and should be specific to the modal.  Could this file be moved all together?